### ISSUE TITLE
Place cursor in Terminal on "Go To Terminal"

### ISSUE THREAD

**Issue by max-winderbaum** [jakob/Postico#138]
When the shift-apple-T hotkey is pressed to "Go To Terminal", the user still needs to click inside the box to place the cursor there. Please place the cursor in the terminal so the user can just start typing.

**Comment by jakob** [comment]
Good point. I'm in the midst of some extensive changes to the SQL editor (along the lines of #18), so I'll see to fix this as well when I get to it.

**Comment by max-winderbaum** [comment]
👍 Your fast and thoughtful response is resulting in me purchasing Postico. Thanks so much!

**Comment by jakob** [comment]
Well, thank *you* for buying a copy :)
